packages/protocol/contracts/verifiers/SgxVerifier.sol

```diff
@@ -149,7 +149,6 @@ contract SgxVerifier is EssentialContract, IVerifier {
         TaikoData.TierProof calldata _proof
     )
         external
-        onlyFromNamed(LibStrings.B_TAIKO)
     {
```

Comment:
I still think it is a security issue as referred to here. https://github.com/taikoxyz/taiko-mono/pull/17960#discussion_r1728417427

Please see SigmaPrime report. 
<img width="971" alt="kép" src="https://github.com/user-attachments/assets/9f80b877-cb0a-4aaa-8159-4d2a85334890">